lib/plugins/aws/package/compile/events/httpApi.js

```diff
@@ -274,15 +274,39 @@ Object.defineProperties(
       const userAuthorizers = userConfig.authorizers;
       const authorizers = (this.config.authorizers = new Map());
       if (userAuthorizers) {
+        if (typeof userAuthorizers !== 'object') {
+          throw new this.serverless.classes.Error(
+            'serverless.yaml is malformed at provider httpApi authorizers - should be an object with names as keys',
+            'MALFORMED_YAML'
+          );
+        }
         if (userConfig.id) {
           throw new this.serverless.classes.Error(
             'Cannot setup authorizers for externally configured HTTP API',
             'EXTERNAL_HTTP_API_AUTHORIZERS_CONFIG'
           );
         }
         for (const [name, authorizerConfig] of Object.entries(userAuthorizers)) {
+          // todo uncomment code when authorizer type is supported
+
+          // if (!authorizerConfig.type) {
+          //   throw new this.serverless.classes.Error(
+          //     `Authorizer '${name}' for httpApi in provider must have type specified`,
+          //     'INVALID_HTTP_API_AUTHORIZER'
+          //   );
+          // }
+          if (
```

Comment:
I believe this check is already implemented as a part of our schema: https://github.com/serverless/serverless/blob/master/lib/plugins/aws/provider.js#L738 Please correct me if I'm wrong

Reply:
If it exists it doesn't work

Reply:
Could you share a reproducible example? I've just tested all required properties (`indentitySource`, `issuerUrl` and `audience`) and if any of them is missing schema validation produces expected warning or error, depending on the chosen mode of validation